android/src/main/java/com/swmansion/gesturehandler/core/LongPressGestureHandler.kt

```diff
@@ -37,6 +41,47 @@ class LongPressGestureHandler(context: Context) : GestureHandler<LongPressGestur
     return this
   }
 
+  fun setNumberOfPointers(numberOfPointers: Int): LongPressGestureHandler {
+    numberOfPointersRequired = numberOfPointers
+    return this
+  }
+
+  private fun getAverageX(ev: MotionEvent, excludePointer: Boolean = false): Float {
+    if (!excludePointer) {
+      return (0 until ev.pointerCount).map { ev.getX(it) }.average().toFloat()
+    }
+
+    var sumX = 0f
+
+    for (i in 0 until ev.pointerCount) {
+      if (i == ev.actionIndex) {
+        continue
+      }
+
+      sumX += ev.getX(i)
+    }
+
+    return sumX / (ev.pointerCount - 1)
+  }
```

Comment:
We can merge `getAverageX` and `getAverageY` into one method. For example, we can return object with `x` and `y`, or pass argument which will determine whether we want to calculate `x` or `y`. We can also leave 2 implementations. 

Let me know which approach do you think is better, @j-piasecki, @latekvo

Reply:
One method makes sense imo, given that now their implementation is exactly the same and we always want both values anyway.

Reply:
I think it's better to have them both consolidated into a single `getAveragePosition` function, as I can't really image them being used separately.

But i see that a similar function already exists [here](https://github.com/software-mansion/react-native-gesture-handler/blob/0ece2902d7853b16a352c31d99ca29a58b0de4bb/android/src/main/java/com/swmansion/gesturehandler/core/GestureUtils.kt#L7), and it follows the separate-function approach.
I see that the built-in `MotionEvent` events also use the separate `getX` and `getY` functions approach.

I'm not sure if it'd be better to follow the convention, as it is not strongly set in our codebase.

Let's wait to see what @j-piasecki thinks.

Reply:
Okay, done in [dc90644](https://github.com/software-mansion/react-native-gesture-handler/pull/3043/commits/dc906440416ed11d14320e0807c1b1cf64ed9309)

Reply:
We may want to consider consolidating other methods that follow this convention

Reply:
> We may want to consider consolidating other methods that follow this convention

We may. I can do this, but definitely in another PR 😅